packages/cli/src/commands/init.ts

```diff
@@ -726,3 +750,21 @@ async function setupEnvironmentVariable(
     logger.success(`✅ Added ${variableName}=${renderer(value)} to ${fileName}`);
   }
 }
+
+async function detectNextJsVersion(projectPath: string): Promise<number> {
```

Comment:
Why is this function being added here?

Reply:
For the log message that'll have the link to the blog where there's a guide on how to complete @trigger.dev/react set up.